### PR TITLE
Bug 1570069: Break out events query into separate tables per doctype

### PR DESCRIPTION
**Pull request by sunahsuh**
Per discussion in slack, I'm breaking these out into separate tables so they can be downstream of their respective copy_dedupe jobs

### CODE REVIEW

File: templates/telemetry_derived/bq_main_events/query.sql [hunk 1]
```diff
@@ -0,0 +1,45 @@
+-- Creates a backwards-compatible flattened events dataset created from gcp-ingested main pings
```
Comment (relud) [inline templates/telemetry_derived/bq_main_events/query.sql]: these tables need version suffixes, e.g. `templates/telemetry_derived/bq_main_events_v1/query.sql`

File: templates/telemetry_derived/bq_main_events/init.sql [hunk 2]
```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.telemetry_derived.bq_main_events` (
```
Comment (relud) [inline templates/telemetry_derived/bq_main_events/init.sql]: these tables need version suffixes
```suggestion
  `moz-fx-data-shared-prod.telemetry_derived.bq_main_events_v1` (
```

Comment (relud) [reply templates/telemetry_derived/bq_main_events/init.sql]: also, i feel like we shouldn't be using a `bq_` prefix here

File: templates/telemetry_derived/bq_event_events/init.sql [hunk 3]
```diff
@@ -0,0 +1,33 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.telemetry_derived.bq_event_events` (
```
Comment (relud) [inline templates/telemetry_derived/bq_event_events/init.sql]: `event_events` oh how i wish there were a better name for this, but alas I can't think of one v_v

File: sql/telemetry_derived/bq_event_events/init.sql [hunk 4]
```diff
@@ -1,5 +1,5 @@
 CREATE TABLE IF NOT EXISTS
-  `moz-fx-data-shared-prod.telemetry_derived.bq_events` (
+  `moz-fx-data-shared-prod.telemetry_derived.bq_event_events` (
```
Comment (jklukas) [inline sql/telemetry_derived/bq_event_events/init.sql]: This table name should have a version suffix.

Comment (jklukas) [reply sql/telemetry_derived/bq_event_events/init.sql]: And why the bq prefix here?

File: sql/telemetry_derived/bq_main_events/init.sql [hunk 5]
```diff
@@ -1,5 +1,5 @@
 CREATE TABLE IF NOT EXISTS
-  `moz-fx-data-shared-prod.telemetry_derived.bq_events` (
+  `moz-fx-data-shared-prod.telemetry_derived.bq_main_events` (
```
Comment (jklukas) [inline sql/telemetry_derived/bq_main_events/init.sql]: I had intended that tables in telemetry_derived are for backend use only and should always be versioned. Then the views we present on top of them in `telemetry` can be unversioned. That gives us more wiggle room to completely replace what the backend tables look like while keeping the user-facing view clean and unburdened with knowing what version of the backend table it's hitting.

File: sql/telemetry_derived/bq_event_events/init.sql (hunk 4 above)
Comment (jklukas) [reply sql/telemetry_derived/bq_event_events/init.sql]: I should have read through @relud's review first. I'm just making extra noise here.

File: templates/telemetry_derived/bq_event_events/init.sql (hunk 3 above)
Comment (sunahsuh) [reply templates/telemetry_derived/bq_event_events/init.sql]: Well, the good news is that users will never see the table name

File: sql/telemetry_derived/bq_event_events/init.sql (hunk 4 above)
Comment (sunahsuh) [reply sql/telemetry_derived/bq_event_events/init.sql]: Re: the bq prefix, we have an existing events table from parquet. I'm open to suggestions but the intention is to differentiate between the old table and the new one

Comment (relud) [reply sql/telemetry_derived/bq_event_events/init.sql]: the existing table is just called `events` though, so this won't collide with that anymore.

**Pull request merged**